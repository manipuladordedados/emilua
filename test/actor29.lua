if _CONTEXT == 'main' then
    spawn_vm('.', { inherit_ctx = false, concurrency_hint = 1 })
else
    assert(_CONTEXT == 'worker')
    spawn_ctx_threads(0)
end
