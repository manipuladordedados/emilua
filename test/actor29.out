Main fiber from VM 0x1 panicked: 'Operation not supported'
stack traceback:
	input.lua:5: in main chunk
	[C]: in function ''
	[string "?"]: in function <[string "?"]:0>
